copyparty/web/splash.js

```diff
@@ -469,6 +469,47 @@ var Ls = {
 		"af1": "mostrar subidas recientes",
 		"ag1": "mostrar usuarios IdP conocidos"
 	},
+	"swe": {
+		"a1": "uppdatera",
+		"b1": "tjena främling &nbsp; <small>(du är inte inloggad)</small>",
+		"c1": "logga ut",
+		"d1": "dumpa stacken",
+		"d2": "visar tillståndet på alla aktiva trådar",
+		"e1": "ladda om konfig.",
+		"e2": "ladda om konfigurationsfiler (konton/volymer/volflaggor),$Noch skanna om alla e2ds-volymer$N$Nobs.: ändrade globala inställningar$Nkräver en fullständig omstart",
+		"f1": "du kan bläddra:",
+		"g1": "du kan ladda upp till:",
+		"cc1": "annat:",
+		"h1": "avaktivera k304",
+		"i1": "aktivera k304",
+		"j1": "med k304 aktiverad kommer klienten att koppla bort sig vid varje HTTP 304-fel, vilket kan hindra vissa buggiga proxyservrar från att fastna (sidor slutar ladda), <em>men</em> saker kommer också att bli långsammare i allmänhet",
+		"k1": "återställ klientinställningar",
+		"l1": "logga in för att se mer:",
+		"m1": "välkommen tillbaka,",
+		"n1": "404 hittades inte &nbsp;┐( ´ -`)┌",
+		"o1": 'eller så har du kanske inte tillgång -- prova ett lösenord eller <a href="' + SR + '/?h">åk hem</a>',
+		"p1": "403 nekat &nbsp;~┻━┻",
+		"q1": 'använd ett lösenord eller <a href="' + SR + '/?h">åk hem</a>',
+		"r1": "åk hem",
+		".s1": "skanna om",
+		"t1": "åtgärd",
+		"u2": "tid sedan senaste serverskrivning$N( uppladdning / namnbyte / ... )$N$N17d = 17 dagar$N1h23 = 1 timme 23 minuter$N4m56 = 4 minuter 56 sekunder",
+		"v1": "koppla upp",
+		"v2": "använd denna server som en lokal disk",
+		"w1": "byt till https",
+		"x1": "byt lösenord",
+		"y1": "redigera utdelningar",
+		"z1": "lås upp denna utdelning:",
+		"ta1": "fyll i ditt nya lösenord",
+		"ta2": "upprepa det nya lösenordet:",
+		"ta3": "det blev fel; vänligen försök igen",
+		"aa1": "inkommande filer:",
+		"ab1": "avaktivera no304",
+		"ac1": "aktivera no304",
+		"ad1": "detta stänger av all cachning; prova detta om k304 inte räckte till. Detta kommer att slösa enorma mängder nätverkstrafik!",
+		"ae1": "aktiva nedladdningar:",
+        "af1": "visa senaste uppladdningar",
+	},
```

Comment:
I think I'm missing a few strings here.

Reply:
ag1 was missing from tl.js but should be there now
https://github.com/9001/copyparty/commit/c2cee222bdc3ae836ca1adb52f2af6381e8a5a1f#diff-759e097addf6e21177148e72a8b1763c84506b38e1614b99cade442eeaab4831